tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "moduleResolution": "bundler",
     "declaration": true,
     "emitDeclarationOnly": true,
+    "jsx": "react-jsx",
```

Comment:
might be better to set this in the react folder, so we don't force an interpretation of JSX to go a certain way